code/game/objects/items/devices/autopsy.dm

```diff
@@ -159,24 +153,9 @@
 	P.info = "<tt>[scan_data]</tt>"
 	P.overlays += "paper_words"
 
-	if(istype(usr,/mob/living/carbon))
-		// place the item in the usr's hand if possible
-		if(!usr.r_hand)
-			P.loc = usr
-			usr.r_hand = P
-			P.layer = 20
-			P.plane = HUD_PLANE
-		else if(!usr.l_hand)
-			P.loc = usr
-			usr.l_hand = P
-			P.layer = 20
-			P.plane = HUD_PLANE
-
-	if(istype(usr,/mob/living/carbon/human))
-		usr:update_inv_l_hand()
```

Comment:
Really, you couldn't even be bothered to typecast. Granted, this has 100% safe, but *really*....***really***.